library/settings.gradle

```diff
@@ -1,2 +1 @@
-rootProject.name = 'library'
-
+rootProject.name = 'odds-feed'
```

Comment:
Good rename... actually, could you also sort out https://github.com/oddin-gg/javasdk/issues/8 (on another PR)? this way odds-feed will be a fully usable library, there'll be no need to host our own fork on Nexus (as we currently do)

Reply:
I see your point, I will discuss it with my bos.